### PR TITLE
add new test lane with modify-vm-template in kubevirt-tekton-tasks

### PR DESCRIPTION
**Pull request by ksimon1**
add new test lane with modify-vm-template in kubevirt-tekton-tasks
Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml [hunk 1]
```diff
@@ -44,6 +44,10 @@ images:
   dockerfile_path: build/wait-for-vmi-status/Dockerfile
   from: openshift_release_golang-1.15
   to: tekton-task-wait-for-vmi-status
+- context_dir: modules/modify-vm-template/
+  dockerfile_path: build/modify-vm-template/Dockerfile
+  from: base_fedora
```
Comment (atiratree) [inline ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml:49]: base_fedora?

Comment (ksimon1) [reply ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml:49]: I am trying to get rid of golang images as we discussed.

Comment (atiratree) [reply ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml:49]: I know, but we do not use fedora there, can't we rather use ubi?

Comment (ksimon1) [reply ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml]: sure, fixed

Comment (atiratree) [reply ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml]: I would rather put the actual image that we use (`ubi-minimal`), even though now it does not seem to make a difference. Not sure why though ...

Comment (ksimon1) [reply ci-operator/config/kubevirt/kubevirt-tekton-tasks/kubevirt-kubevirt-tekton-tasks-main.yaml]: fixed

**Comment by atiratree** [comment]
@ksimon1 could you please do this for other golang images that we depracated? - to be consistent

**Comment by ksimon1** [comment]
Sure thing, I was waiting for this PR

**Pull request merged**